core/src/test/java/org/apache/spark/JavaAPISuite.java

```diff
@@ -25,17 +25,17 @@
 import java.util.concurrent.*;
 
 import org.apache.spark.input.PortableDataStream;
+import org.apache.spark.rdd.RDD;
+import org.apache.spark.serializer.KryoSerializer;
+import scala.collection.JavaConversions;
 import scala.Tuple2;
 import scala.Tuple3;
 import scala.Tuple4;
 
-import com.google.common.collect.Iterables;
-import com.google.common.collect.Iterators;
-import com.google.common.collect.Lists;
-import com.google.common.collect.Maps;
 import com.google.common.base.Throwables;
 import com.google.common.base.Optional;
 import com.google.common.base.Charsets;
+import com.google.common.collect.*;
```

Comment:
I'd reverse this ... I don't know whether we have a strong convention either way about * and _ imports but I wouldn't change it just to change it. Generally * is avoided in Java.